include/dsn/service_api_c.h

```diff
@@ -500,7 +500,7 @@ extern DSN_API bool         dsn_semaphore_wait_timeout(
 //------------------------------------------------------------------------------
 
 // rpc address utilities
-
+extern DSN_API int           dsn_address_use_ip_as_name;
```

Comment:
It is a little tricky here. If we put the declaration of "use_ip_as_name" global variable in address.h(and more, it is used in rpc_address.h), but put the definition in address.cpp, then the variable is visible by users (address.h is public), but the definition symbol is in "libdsn.core.so" and not visible.